<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="BatteryWidget" name="batteryWidget" native="true">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>50</y>
          <width>211</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditDisplay">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>201</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>251</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Add Battery</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineCapacity">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Capacity (mAh)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineVoltage">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Voltage (V)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddBattery">
           <property name="text">
            <string>Add Battery</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Create Pack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboPackType">
           <item>
            <property name="text">
             <string>SERIES</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PARALLEL</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCreatePack">
           <property name="text">
            <string>Create Pack</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUsePack">
           <property name="text">
            <string>Use Battery</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRechargePack">
           <property name="text">
            <string>Recharge Battery</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatteryWidget</class>
   <extends>QWidget</extends>
   <header location="global">batterywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
